splinterd/src/main.rs

```diff
@@ -267,6 +333,16 @@ fn main() {
                 .multiple(true)
                 .alias("peer"),
         )
+        .arg(
+            Arg::with_name("peering_key")
+                .long("peering-key")
+                .long_help(
+                    "The name of the key to use for challenge authorization with specified peers. \
```

Comment:
Argument help should ideally fit on a single line.

Reply:
Shortened the help line